File: src/main/java/toast/pine/Adapters/AdapterManager.java
package toast.pine.Adapters;

public class AdapterManager {

    private final MonsterAdapter monsterAdapter = new MonsterAdapter();
    private final MonsterTypeAdapter monsterTypeAdapter = new MonsterTypeAdapter();
    private final ItemAdapter itemAdapter = new ItemAdapter();
    private final ItemMaterialAdapter itemTypeAdapter = new ItemMaterialAdapter();


    public MonsterAdapter getMonsterAdapter() {
        return monsterAdapter;
    }


    public MonsterTypeAdapter getMonsterTypeAdapter() {
        return monsterTypeAdapter;
    }

    public ItemAdapter getItemAdapter() {
        return itemAdapter;
    }

    public ItemMaterialAdapter getItemMaterialAdapter() {
        return itemTypeAdapter;
    }

}

File: src/main/java/pine/toast/toastrpg/core/adapters/AdapterManager.kt
package pine.toast.toastrpg.core.adapters

import pine.toast.toastrpg.core.ToastRPG

class AdapterManager {
    val monsterAdapter = MonsterAdapter()
    val monsterTypeAdapter = MonsterTypeAdapter()
    val itemAdapter = ItemAdapter()
    val itemMaterialAdapter = ItemMaterialAdapter()
    val socialProfileAdapter = SocialProfileAdapter()
    val skillAdapter = SkillAdapter()

    init {
        ToastRPG.getPassedPlugin()!!.logger.info(" - AdapterManager ~ Loaded")
    }
}
